components/suggest/src/rs.rs

```diff
@@ -141,6 +141,11 @@ impl From<RecordRequest> for GetItemsOptions {
         if let Some(last_modified) = value.last_modified {
             options.filter_gt("last_modified", last_modified.to_string());
         }
+
+        if let Some(exposure_suggestion_type) = value.exposure_suggestion_type {
+            options.filter_eq("exposure_suggestion_type", exposure_suggestion_type);
```

Comment:
I think this is going to conflict with some of my recent PR -- sorry about that.  Also, the general advice we've gotten from the RS team is to not use any filters when querying the data and do all filtering client-side.